docs/src/getting-started/authorization-policies.md

```diff
@@ -56,7 +56,6 @@ allow if
   resource($res),
   operation($op),
   right($res, $op);
-
```

Comment:
empty lines mess with how markdown renders inline html, sadly. I haven't found a clean way to tell markdown: "this whole block is HTML, don't try to parse it, just output it as is"